kork/kork-artifacts/src/main/java/com/netflix/spinnaker/kork/artifacts/artifactstore/s3/S3ArtifactStoreConfiguration.java

```diff
@@ -72,20 +77,25 @@ public ArtifactStoreGetter artifactStoreGetter(
   }
 
   @Bean
-  public S3Client artifactS3Client(ArtifactStoreConfigurationProperties properties) {
-    S3ClientBuilder builder = S3Client.builder();
+  public S3Client artifactS3Client(
```

Comment:
This feels like a separate change from the entity store.  Is it?  What does it do?   Worth separating into its own commit, potentially with a test?

Reply:
This commit was old, and I didn't want to dissect the difference. I believe open source handled custom credentials to AWS differently than how we implemented it internally, but I know we've tested our change a lot internally, so I went with ours, but I can revert if needed.